etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/CreateRawBaseTables.java

```diff
@@ -324,7 +324,8 @@ private void checkAndCreateBaseTable(String dbName, String baseTable, String ddl
 
             Connection con = getHiveJDBCConnection(dbName);
             Statement stmt = con.createStatement();
-            ResultSet rs = stmt.executeQuery("SHOW TABLES LIKE '" + baseTable + "'");
+            String bTable = baseTable;
+            ResultSet rs = stmt.executeQuery("SHOW TABLES LIKE '" + bTable + "'");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) "baseTable" is provided externally to the method and not sanitized before use. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2077)